tests_my/pages/dashboard/dashboard_page.py

```diff
@@ -0,0 +1,93 @@
+from tests_my.testcases.utils import constants, browser_helper
+from selenpy.element.base_element import BaseElement
+from selenpy.element.text_box import TextBox
+from selenpy.element.combo_box import ComboBox
+from selenpy.element.check_box import CheckBox
+
+
+class DashboardPage():
+
+    def __init__(self):
+        self._dashboard = BaseElement("id=div_main_body")
+        self._lbl_user_menu = BaseElement("//a[@href='#Welcome']")
+        self._lbl_logout = BaseElement("//a[contains(@href,'logout')]")
+        self._lbl_setting = BaseElement("class=mn-setting")
+        self._lbl_item_setting = BaseElement("//li[@class='mn-setting']//a[normalize-space()='%s']")
+        self._lbl_menu_setting = BaseElement("//span[normalize-space()='Global Setting']")
+        self._lbl_page = BaseElement("//a[normalize-space()='%s']")
+        self._txt_name_page = TextBox("css=#div_popup #name")
+        self._cbb_parent_page = ComboBox("css=#parent")
+        self._btn_ok = BaseElement("css=#OK")
+        self._lbl_pop_up = BaseElement("css=#div_popup h2")
+        self._cb_public = CheckBox("css=#div_popup #ispublic")
+        self._cbb_number_column = ComboBox("css=#div_popup #columnnumber")
+        self._cbb_display_after = ComboBox("css=#div_popup #afterpage")
+
+    def is_At(self):
+        return self._dashboard.is_displayed(constants.WAIT_CONDITION_TIME)
+
+    def log_out(self):
+        if self._dashboard.is_displayed():
+            self._lbl_user_menu.click()
+            self._lbl_logout.click()
+
+    def get_message(self):
```

Comment:
update the meaningful name for this. what's the message that you want to get?

Reply:
I updated